ansible-tests/validations/neutron-sanity-check.yaml

```diff
@@ -8,6 +8,33 @@
         Neutron's configuration.
       groups:
         - post-deployment
+    config_files:
+    - /etc/neutron/neutron.conf
+    - /usr/share/neutron/neutron-dist.conf
+    - /etc/neutron/metadata_agent.ini
+    - /etc/neutron/metering_agent.ini
+    - /etc/neutron/dhcp_agent.ini
+    - /etc/neutron/plugins/ml2/openvswitch_agent.ini
+    - /etc/neutron/conf.d/ml2_conf_cisco.ini
+    - /etc/neutron/conf.d/cisco_cfg_agent.ini
+    - /etc/neutron/conf.d/cisco_router_plugin.ini
+    - /etc/neutron/dhcp_agent.ini
+    config_dirs:
+    - /usr/share/neutron/l3_agent
+    - /etc/neutron/conf.d/common
+    - /etc/neutron/conf.d/neutron-l3-agent
+    - /usr/share/neutron/neutron-lbaas-dist.conf
```

Comment:
This looks more like a config file to me. Same for `/etc/neutron/lbaas_agent.ini` below.